### PR TITLE
action: fix cocoa pods build version

### PR DESCRIPTION
**Pull request by lvaccaro**
rename version name from lwk `bindings_x.x.x` to `x.x.x`

### CODE REVIEW

File: .github/workflows/publish-swift-package.yml [hunk 1]
```diff
@@ -41,30 +41,35 @@ jobs:
...
+          echo "VERSION=$(echo ${{ env.RELEASE_REF }} | sed 's/bindings_//')" >> $GITHUB_ENV
       - name: Update Swift Package definition
         run: |
           sed -i '' 's#.binaryTarget(name: "lwkFFI".*$#.binaryTarget(name: "lwkFFI", url: "https://github.com/Blockstream/lwk-swift/releases/download/${{ inputs.version }}/lwkFFI.xcframework.zip", checksum: "${{ env.XCF_CHECKSUM }}"),#' Package.swift
       - name: Update Cocoapods definitions
         run: |
-          sed  -i '' 's/^.\{4\}spec.version.*$/    spec.version                = "${{ inputs.version }}"/g' lwkFFI.podspec
-          sed  -i '' 's/^.\{4\}spec.version.*$/    spec.version                = "${{ inputs.version }}"/g' LiquidWalletKit.podspec
+          sed  -i '' 's/^.\{4\}spec.version.*$/    spec.version                = "${{ env.VERSION }}"/g' lwkFFI.podspec
+          sed  -i '' 's/^.\{4\}spec.source.*$/     spec.source                = { :http => "https://github.com/Blockstream/lwk-swift/releases/download/${{ inputs.version }}/lwkFFI.xcframework.zip" }/g' lwkFFI.podspec
```
Comment (RCasatta) [inline .github/workflows/publish-swift-package.yml]: is `inputs.version` here correct?

File: .github/workflows/publish-swift-package.yml [hunk 2]
```diff
@@ -41,30 +41,35 @@ jobs:
...
+      - name: Set release version
+        run: |
+          echo "RELEASE_REF=${GITHUB_REF#refs/*/}" >> $GITHUB_ENV
+          echo "VERSION=$(echo ${{ env.RELEASE_REF }} | sed 's/bindings_//')" >> $GITHUB_ENV
       - name: Update Swift Package definition
         run: |
           sed -i '' 's#.binaryTarget(name: "lwkFFI".*$#.binaryTarget(name: "lwkFFI", url: "https://github.com/Blockstream/lwk-swift/releases/download/${{ inputs.version }}/lwkFFI.xcframework.zip", checksum: "${{ env.XCF_CHECKSUM }}"),#' Package.swift
       - name: Update Cocoapods definitions
         run: |
-          sed  -i '' 's/^.\{4\}spec.version.*$/    spec.version                = "${{ inputs.version }}"/g' lwkFFI.podspec
```
Comment (RCasatta) [inline .github/workflows/publish-swift-package.yml:49]: can we add an initial 
 `echo Version:${{ env.VERSION }}`

**Review by RCasatta — APPROVED** [review]
utACK 81cbcfcea8e462a56eb4284d0263e7ab46f12580

File: .github/workflows/publish-swift-package.yml (hunk 1 above)
Comment (lvaccaro) [reply .github/workflows/publish-swift-package.yml]: right, fixed

**Pull request merged**